### ISSUE TITLE
Missing extra points

### ISSUE THREAD

**Issue by guga31bb** [ryurko/nflscrapR-data#5]
GameID: 2017102908

Earl Thomas scores a TD on an INT return and Seattle's following extra point is not included in the play by play data. The score is then wrong for the remainder of the game. This may affect the WP calculations too.

The same thing happened in SEA-IND when Seattle had a pick-6 in that game, so it might be related to defensive scores (GameID 2017100113).

**Comment by ryurko** [comment]
Yeah this is an NFL issue since they just don't upload the PAT on returns to the play-by-play, unfortunately don't have a fix at the moment.

**Comment by jacole3** [comment]
> Yeah this is an NFL issue since they just don't upload the PAT on returns to the play-by-play, unfortunately don't have a fix at the moment.

I recently came across the same issue when looking back at past seasons of the NFLScrapR data. Based on a sample size of a few defensive TDs I looked at, it seemed like the ensuing PAT would never be included from 2009-2017, but would be included properly since 2018. (Some examples: game_id == 2018111900 and game_id == 2019111009)

Did the league change the way it uploads plays entering the 2018 season? And if so, is there no way we can get the pre-2018 data to show up the right way?
